packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/ArbTokenBridge.sol

```diff
@@ -41,6 +46,8 @@ contract ArbTokenBridge is CloneFactory {
     ICloneable public immutable templateERC777;
     address public immutable l1Pair;
 
+    event MintAndCallTriggered(bool success);
```

Comment:
We might want other info in this event?

Reply:
On it

Reply:
Do we want to also emit the data used in the call? I added in sender, dest and amount.